ports/cpp-exiftool/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.20)

```

```suggestion
cmake_minimum_required(VERSION 3.21)
```

We already require that based on https://github.com/microsoft/vcpkg/blob/d33fbc3dde2ab2cbba7e7137c9c0d0780e83627a/scripts/ports.cmake#L1